src/main/scala/com/codacy/plugins/api/package.scala

```diff
@@ -8,6 +8,8 @@ import com.codacy.plugins.api.results.{Parameter, Pattern, Result, Tool}
 
 package object api {
 
+import scala.collection.immutable.SortedSet
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/gh/codacy/codacy-engine-scala-seed/pullRequest?prid=5235671)